lib/cli.js

```diff
@@ -236,6 +239,10 @@ const meowOptions = {
       --allow-empty-input, --aei
 
         When glob pattern matches no files, the process will exit without throwing an error.
+	 
+	  --globby-options, --go 
+		
+	    CLI option to traverse dotfiles i.e Hidden folders files will not be ignored(e.g. stylelint --fix '**/*.css)
```

Comment:
[suggest] How about simplifying the description?

```suggestion
	    Options in JSON format passed to globby.
```

Also, let's remove extra trailing whitespace and convert TABs to whitespace. If doing so, we can get less diff in `cli.test.js.snap`.